src/test/FuzzerImpl.cpp

```diff
@@ -419,17 +596,44 @@ getFuzzConfig(int instanceNumber)
 }
 
 static void
-resetRandomSeed()
+resetRandomSeed(uint seed = 1)
```

Comment:
do not use default arguments: we're dealing with a single file here, so it's not like we can't track down all the places this is called from

Reply:
I've removed all the default arguments.